org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/rename/ChangeConverter.xtend

```diff
@@ -31,21 +35,35 @@ class ChangeConverter implements IAcceptor<IEmfResourceChange> {
 
 	static class Factory {
 
-		@Inject IResourceServiceProvider.Registry registry
+		@Inject protected IResourceServiceProvider.Registry registry
 
+		/**
+		 * @deprecated use {@link #create(WorkspaceManager, WorkspaceEdit, boolean}�instead.
```

Comment:
Looks like the link points to a method that does not exist

Reply:
Fixed